shell/platform/windows/client_wrapper/flutter_engine_unittests.cc

```diff
@@ -121,4 +129,21 @@ TEST(FlutterEngineTest, GetMessenger) {
   EXPECT_NE(engine.messenger(), nullptr);
 }
 
+TEST(FlutterEngineTest, DartEntrypointArgs) {
+  DartProject project(L"data");
+  std::vector<std::string> arguments(2);
+  arguments[0] = "one";
+  arguments[1] = "two";
+  project.set_dart_entrypoint_arguments(arguments);
+  testing::ScopedStubFlutterWindowsApi scoped_api_stub(
+      std::make_unique<TestFlutterWindowsApi>());
+  auto test_api = static_cast<TestFlutterWindowsApi*>(scoped_api_stub.stub());
+
+  FlutterEngine engine(project);
+  const std::vector<std::string>& arguments_ref = test_api->dart_entrypoint_arguments();
+  EXPECT_TRUE(arguments[0] == arguments_ref[0]);
+  EXPECT_TRUE(arguments[1] == arguments_ref[1]);
+  EXPECT_EQ(2, arguments_ref.size());
```

Comment:
Nit: put this first and ASSERT_EQ so if for some reason it's wrong the test won't crash.